core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/results/ValidationResult.java

```diff
@@ -189,6 +189,13 @@ public final Resource getId() {
 		return id;
 	}
 
+	/**
+	 * @return conforms
+	 */
+	public boolean conforms() {
+		return severity != Severity.Violation;
+	}
```

Comment:
It never really made sense to support other severity levels before, since anything lower then sh:Violation wouldn't result in a validation exception. Now that we return a validation report regardless I've taken the time to implement support for sh:Warning and sh:Info.